app/containers/message/index.test.tsx

```
@@ -0,0 +1,69 @@
+			loadAsync = jest.fn();
+			playAsync = jest.fn();
+			unloadAsync = jest.fn();
+		},
+		InterruptionModeIOS: { MixWithOthers: 0, DoNotMix: 1 },
+		InterruptionModeAndroid: { MixWithOthers: 0, DoNotMix: 1 },
+		requestPermissionsAsync: jest.fn(() => Promise.resolve({ status: 'granted', granted: true, canAskAgain: true })),
+		getPermissionsAsync: jest.fn(() => Promise.resolve({ status: 'granted', granted: true, canAskAgain: true }))
+	};
+});
+
+import { MessageContainer } from './index';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)